test/uk/gov/hmrc/partnershipidentificationfrontend/service/AuditServiceSpec.scala

```diff
@@ -326,83 +336,116 @@ class AuditServiceSpec extends AnyWordSpec with Matchers with IdiomaticMockito {
+            limitedPartnershipFullJourneyData
+          ))
+
+          await(TestAuditService.auditPartnershipInformation(
+            journeyId = testJourneyId,
+            journeyConfig = journeyConfig
+          ))
+
+          mockAuditConnector.sendExplicitAudit(
+            auditType = "ScottishLTDPartnershipRegistration",
+            detail = expectedAuditModel
+          ) was called
+        }
+      }
+
+      "identifiersMatch is false and service name has not provided" should {
+        "audit the correct information" in new Setup {
+          mockPartnershipIdentificationService.retrievePartnershipFullJourneyData(testJourneyId) returns Future.successful(Some(
+            limitedPartnershipFullJourneyData
+          ))
+
+          mockAppConfig.defaultServiceName returns testServiceName
 
-        mockAuditConnector.sendExplicitAudit(
-          "LimitedLiabilityPartnershipRegistration",
-          expectedAuditModel
-        ) was called
+          await(TestAuditService.auditPartnershipInformation(
+            journeyId = testJourneyId,
+            journeyConfig = journeyConfig.copy(pageConfig = testDefaultPageConfig.copy(optServiceName = None))
+          ))
+
+          mockAuditConnector.sendExplicitAudit(
+            auditType = "ScottishLTDPartnershipRegistration",
+            detail = expectedAuditModel
+          ) was called
+        }
       }
     }
   }
 
-  val limitedPartnershipFullJourneyData: PartnershipFullJourneyData = PartnershipFullJourneyData(
-    optPostcode = Some(testPostcode),
-    optSautr = Some(testSautr),
-    companyProfile = Some(testCompanyProfile),
-    identifiersMatch = false,
-    businessVerification = Some(BusinessVerificationUnchallenged),
-    registrationStatus = RegistrationNotCalled
+  private def expectedNonLimitedPartnershipAuditJson(partnershipType: String,
+                                                     verificationStatus: String,
+                                                     registerStatus: String,
+                                                     isMatch: Boolean = true): JsObject = Json.obj(
+    "SAUTR" -> testSautr,
+    "SApostcode" -> testPostcode,
+    "isMatch" -> isMatch,
+    "businessType" -> partnershipType,
+    "VerificationStatus" -> verificationStatus,
+    "RegisterApiStatus" -> registerStatus,
+    "callingService" -> testServiceName
   )
 
-  private def expectedAuditJson(partnershipType: String, verificationStatus: String, registerStatus: String, isMatch: Boolean = true): JsObject =
-    Json.obj(
-      "SAUTR" -> testSautr,
-      "SApostcode" -> testPostcode,
-      "isMatch" -> isMatch,
-      "businessType" -> partnershipType,
-      "VerificationStatus" -> verificationStatus,
-      "RegisterApiStatus" -> registerStatus,
-      "callingService" -> testServiceName
-    )
+  private def expectedLimitedPartnershipAuditJson(businessType: String): JsObject = expectedNonLimitedPartnershipAuditJson(
```

Comment:
just for consistency can we change it to partnershipType?

Reply:
of course .... suggestion approved!!!